app/src/config/settings/prod.py

```
@@ -0,0 +1,36 @@
+ALLOWED_HOSTS = ["comicchase.com.tw"]
+
+# CSRF settings
+CSRF_TRUSTED_ORIGINS = [
+    "http://comicchase.com.tw",
+    "https://comicchase.com.tw",
+]
+
+DATABASES = {
+    "default": {
+        "ENGINE": "django.db.backends.postgresql",
+        "NAME": config("POSTGRES_DB", default="comic_db"),
```

Ruff (0.14.7)

`config` may be undefined, or defined from star imports

(F405)